frontend/src/components/ChatComponents/RecieverMessage.js

```diff
@@ -42,9 +42,9 @@ export default function RecieverMessage({
             ></Avatar>
           </Tooltip>
         )}
-        <div className="bg-[#FFFFFF]  rounded-tr-xl ml-[1%] font-Roboto rounded-br-xl rounded-bl-xl box-border px-2 py-2 max-[900px]:text-sm flex justify-between">
```

Comment:
Seems like you are also changing the message styles ! Is it necessary ?

Reply:
Yes @Spiral-Memory  @SamarthKadam request to make the message similar to whatsapp web So i changed the message style